website/src/hooks/nft.ts

```diff
@@ -0,0 +1,27 @@
+import { Address } from "viem";
+import { hemi } from "viem/chains";
+import { useReadContract } from "wagmi";
+
+const abi = [
```

Comment:
ABI could also be obtained from the minter package.

Reply:
Done here [881bad5](https://github.com/hemilabs/tge-mint-page/pull/5/commits/881bad5772898df57965dc7f5773d75adfde699f)